src/Mod/Sketcher/Gui/TaskSketcherMessages.cpp

```diff
@@ -41,30 +41,18 @@ using namespace SketcherGui;
 using namespace Gui::TaskView;
 namespace bp = boost::placeholders;
 
-TaskSketcherMessages::TaskSketcherMessages(ViewProviderSketch *sketchView) :
-    TaskBox(Gui::BitmapFactory().pixmap("document-new"), tr("Solver messages"), true, nullptr),
+TaskSketcherMessages::TaskSketcherMessages(ViewProviderSketch *sketchView, QWidget* parent) :
     sketchView(sketchView),
     ui(new Ui_TaskSketcherMessages)
 {
-    // we need a separate container widget to add all controls to
-    proxy = new QWidget(this);
-    ui->setupUi(proxy);
-    QMetaObject::connectSlotsByName(this);
+    ui->setupUi(this);
 
-    this->groupLayout()->addWidget(proxy);
+    QMetaObject::connectSlotsByName(this);

```

Comment:
This will be already called by `setupUi`

Reply:
Still present in the code

Reply:
You meant `QMetaObject::connectSlotsByName(this);` right?
No it isn't, or I don't think so, see the pr modifications : 
 https://github.com/FreeCAD/FreeCAD/pull/7581/files#diff-8c7e8f931ea42cf16456002d4c4150946295c03832e7ad8ac0906b17f542e659

Reply:
My bad :) Sorry